src/Form/Form.test.js

```diff
@@ -96,4 +99,22 @@ describe('Form', () => {
     expect(changedRoundOne).toBeInTheDocument();
   });
 
+  it('should fire the handleSubmit function when the button is clicked', () => {
+    const { getByRole } = render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <Form />
+        </Provider>
+      </BrowserRouter>);
+      
+    //I think this test requires thunk testing
```

Comment:
You might be able to just provide your test store a think middleware, then await the assertion on line 116. I'm not sure, could be worth a try, though.